### PR TITLE
Created Orbital Helper UI

### LINKED ISSUE #140: Orbital Helper UI
Create UI to help with viewing scheduled contacts and performing health checks.

### LINKED ISSUE #148: Health check packge
- Add interfaces and implementation of health checks to `@azure/orbital-integration-common`. 
-  Refactor Orbital Integration UI to use these shared functions instead of the existing code in `health.tsx`.

Eventually one or more health canary jobs will also use these shared functions.

### PR DESCRIPTION
**Pull request by micholas**
# Description
Fixes #140
Fixes #148

## Dependencies affected:
-  n/a

## Checklist before merging
- [x] Properly labeled PR 
- [x] Licensing statement added to new files 
- [x] Downstream dependencies have been addressed
- [x] Corresponding changes to the documentation have been made
- [x] Issue is linked under the development section

### CODE REVIEW

**Comment by scschneider** [comment]
Instead of making comments on each occurrence, I will call it out here. I noticed a hard dependency on arm-orbital-helper in the tcp-to-blob and orbital-helper-ui. There should be a common space for shared dependencies and since each of these 3 projects are unique and provide completely different functionality from one-another, the common space should not be arm-orbital-helper.

**Comment by micholas** [comment]
> Instead of making comments on each occurrence, I will call it out here. I noticed a hard dependency on arm-orbital-helper in the tcp-to-blob and orbital-helper-ui. There should be a common space for shared dependencies and since each of these 3 projects are unique and provide completely different functionality from one-another, the common space should not be arm-orbital-helper.

Good point. I created a `common-js` package and refactored to move appropriate code there.

File: tools/orbital-helper-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. Ensure [logged in](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli) to Azure CLI and default
+   subscription is set.
+    1. `az login` (see [docs](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli))
+    2. `az account set -s "${SUBSCRIPTION_ID}"` (
+       see [docs](https://docs.microsoft.com/en-us/cli/azure/manage-azure-subscriptions-azure-cli#change-the-active-subscription))
+2. From `azure-orbital-integration` directory: `yarn install && yarn build-tools`
+
+## Run
+
+`yarn start` or `yarn start-dev` (to re-build and run)
```
Comment (scschneider) [inline tools/orbital-helper-ui/README.md:35]: This needs to run from `tools/orbital-helper-ui`. From step 2 above, we are in the root of `azure-orbital-integration`.

Comment (micholas) [reply tools/orbital-helper-ui/README.md]: Addressed, thanks!

**Review by scschneider — APPROVED** [review]

**Pull request merged**